### ISSUE TITLE
The request in geckos({authorization}) is undefined

### ISSUE THREAD

**Issue by dsteinman** [geckosio/geckos.io#43]
```
const io = geckos({
	ordered: false, // in a game, this should be set to false (default is false)
	maxRetransmits: 0, // in a game, this should be set to 0 (default is 0)
	iceServers: process.env.NODE_ENV === 'production' ? iceServers : [],
	authorization: async (auth, request) => {
		console.log('request', request);
	}
})
```

Unfortunately I'm getting an undefined for the `request` object:

```
request undefined
```

 I'm wondering if this could be a bug.  It's not used in the examples (chat, phaser3).

(great project otherwsie btw!)

**Comment by yandeu** [comment]
This feature will only be available in  > v1.6.2. 
Geckos.io is now at v1.6.1.

It should be available within the next 10 days.

---

`(great project otherwsie btw!)`
Thx 😁

**Comment by dsteinman** [comment]
Okay, no problem, I figured it was something simple.
